### ISSUE TITLE
Undefined function or variable 'openVesselLabels'

### ISSUE THREAD

**Issue by soulnd** [ignaciorlando/fundus-vessel-segmentation-tbme#5]
1. When I run the script_experiments_massive according to the user_manual, there is a error that matlab can not find the openVesselLabels . 
2. I don't know how to run on my dataset because my dataset don't have binary ground truth masks of the vessels. Can I use DRIVE to train and validate ,then use my dataset to test?

**Comment by ignaciorlando** [comment]
Hi @soulnd. Sorry for not answering before, I was on holidays.

1. It could be possible that I accidentally removed that function. While I try to reproduce the bug, would you mind to checkout to the branch "refactoring" and see if the problem persists? That branch contains a newer version of the project that should be more stable and easy to use, and it will become master after testing.

2. If you don't have binary labels for your test set, you can put all your images on DRIVE/test, for example, to segment vessels from your data set with a model trained on DRIVE. Please, take into account that if there are huge changes in the resolution then you won't get proper results due to feature parameters. The easiest way to deal with that issue would be to downsample your images to approximately the same resolution on DRIVE, or to multiply all feature parameters by a scaling factor as the one we reported in our paper (\rho).

Let me know if (1) works. I'll try to reproduce the bug in the meantime.

Cheers,

Nacho

**Comment by soulnd** [comment]
I download the branch "master"  you revised and run the code again, the problem is solved. But in your  branch "refactoring", the file openVesselLabels.m is still not found.

**Comment by ignaciorlando** [comment]
Hi @soulnd 

The refactoring branch should be working now (at least is working good in my laptop). The openVesselLabels file was always in the fundus-util folder, but the issue was at the time of computing the quality values in the validation set. Let me know if it works in your computer.

Cheers,

Nacho

**Comment by soulnd** [comment]
I run the code again，and it works.

**Comment by ignaciorlando** [comment]
Great!

**Comment by janasubrata** [comment]
script_experiments_massive
Undefined function or variable 'DRIVE'.

Error in getConfiguration_GenericDataset (line 15)
    config.dataset = DRIVE;
how to solve this problem

**Comment by janasubrata** [comment]
script_experiments_massive
Undefined function or variable 'DRIVE'.

Error in getConfiguration_GenericDataset (line 15)
    config.dataset = DRIVE;
I.m uing windows platform matlab 14
